src/frontend/src/views/Upload/components/Metadata/components/ImportFile/parseFile.ts

```diff
@@ -117,13 +117,13 @@ function buildMetadata({ headers, row, warningMessages }: RowInfo) {
   ) {
     metadata.collectionLocation = "";
   }
-
   return metadata;
 }
 
-function convertValue(value: string) {
-  if (value === "Yes") return true;
-  if (value === "No") return false;
+function convertValue(key: string, value: string) {
+  if (key === "keepPrivate" && value === "") return true;
+  if (value.toUpperCase() === "YES") return true;
+  if (value.toUpperCase() === "NO") return false;
 
   return value;
```

Comment:
And then as the final return can we do `return False` for the default again. Since right now what happens is if a user input garbage like "asdasda" it will pass all these cases and then that get's passed to the backend